tests/chainer_tests/functions_tests/loss_tests/test_softmax_cross_entropy.py

```diff
@@ -60,10 +65,12 @@ def check_forward(self, x_data, t_data, class_weight, use_cudnn='always'):
         with chainer.using_config('use_cudnn', use_cudnn):
             loss = functions.softmax_cross_entropy(
                 x, t, normalize=self.normalize,
-                cache_score=self.cache_score, class_weight=class_weight)
+                cache_score=self.cache_score, class_weight=class_weight,
+                enable_double_backprop=self.enable_double_backprop)
         self.assertEqual(loss.data.shape, ())
         self.assertEqual(loss.data.dtype, self.dtype)
-        self.assertEqual(hasattr(loss.creator, 'y'), self.cache_score)
+        if not self.enable_double_backprop:
+            self.assertEqual(hasattr(loss.creator, 'y'), self.cache_score)
         loss_value = float(cuda.to_cpu(loss.data))
```

Comment:
How about adding a test that checks if the forward outputs with and without `enable_double_backprop` are almost equal to each other? (or checking both of them against some naive implementation (e.g. using only numpy interface) is also ok).

Reply:
Added tests that check consistency of two implementations.